.buildkite/updates.yml

```diff
@@ -0,0 +1,16 @@
+steps:
+  - label: 'Update hackage.nix'
+    command:
+      - nix-build -I nixpkgs=https://github.com/NixOS/nixpkgs-channels/archive/nixos-18.09.tar.gz -A maintainer-scripts.update-hackage -o update-hackage.sh
```

Comment:
not sure if supported by the nix version actually use, but there is this syntax:
`-I nixpkgs=channel:nixos-18.09`

Reply:
Thanks, fixed.